### PR TITLE
fixes #26554 - move class_edit to webpack.

### PR DESCRIPTION
**Pull request by ohadlevy**
this is a pure move  + lint, no code changes introduced.

### CODE REVIEW

**Review by Ron-Lavi — DISMISSED** [review]
Thanks @ohadlevy Looks safe to merge :+1:

**Comment by ohadlevy** [comment]
@tbrisker would you consider merging this? it looks low prio to me and speed up some initial page loads in the UI.

**Review by tbrisker — DISMISSED** [review]
Didn't test it but fine with merging this week if someone tests to verify.

**Comment by ohadlevy** [comment]
anyone willing to test/verify? :-)

**Comment by ohadlevy** [comment]
> Thanks @ohadlevy Looks safe to merge

did you actually test this? if so this might be OK to merge based on @trisker comment?

**Comment by Ron-Lavi** [comment]
sorry @ohadlevy I don't have the right environment yet, @tbrisker could you test it live ?

File: webpack/assets/javascripts/foreman_class_edit.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    .clone();
+  content.attr('id', `selected_puppetclass_${id}`);
+  content.append(
+    `<input id='${type}_puppetclass_ids_' name='${type}[puppetclass_ids][]' type='hidden' value=${id}>`
+  );
+
+  const link = content.children('a');
+  const links = content.find('a');
+
+  links.attr('onclick', 'removePuppetClass(this)');
```
Comment (xprazak2) [inline webpack/assets/javascripts/foreman_class_edit.js]: Getting error on puppet class removal, change to `tfm.classEditor.removePuppetClass(this)` fixes the problem.

remove-class

File: webpack/assets/javascripts/foreman_class_edit.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  content.append(
+    `<input id='config_group_ids' name=${type}[config_group_ids][] type='hidden' value=${id}>`
+  );
+  $(`#selected_config_group_${id}`).show('highlight', 5000);
+  $(`#config_group_${id}`)
+    .addClass('selected-marker')
+    .hide();
+  const link = content.children('a');
+  const links = content.find('a');
+  link.attr('onclick', 'removeConfigGroup(this)');
```
Comment (xprazak2) [inline webpack/assets/javascripts/foreman_class_edit.js]: Same as above.

**Comment by ohadlevy** [comment]
thanks @xprazak2 ! updated and rebased.

**Review by xprazak2 — APPROVED** [review]
Works now.

**Comment by ohadlevy** [comment]
@xprazak2 ready to merge than? :)

**Comment by xprazak2** [comment]
Yes, I did not merge because we are close to branching. If we want this for 1.22, @tbrisker can do the honors :)

**Comment by tbrisker** [comment]
Thanks @ohadlevy !

**Pull request merged**